cpmpy/tools/dimacs.py

```diff
@@ -101,29 +86,22 @@ def read_dimacs(fname, sep=None):
         cnf = "\n".join(lines[i+1:]) # part of file containing clauses
```

Comment:
It is mentioned that "If the header is omitted in the file, the number of variables and constraints are inferred."

But we basically do not allow it to be omitted in the current version, so one of the 2 should change

Reply:
We do infer the number of variables and constraints, but the comment should be worded more carefully. The header line should always be present (that is, somewhere in the file you should have `p cnf`) but it does not necessarily need to include the number of variables and consrtaints